tests/integration/v3_lease_test.go

```diff
@@ -142,26 +143,84 @@ func TestV3LeaseGrantByID(t *testing.T) {
 	}
 }
 
-// TestRejectLeaseRequestWithInvalidID ensures leases request with a negative ID is rejected
-func TestRejectLeaseRequestWithInvalidID(t *testing.T) {
+// TestV3LeaseGrantByNegativeID ensures leases may be created by a negative given id and handled correctly
+func TestV3LeaseGrantByNegativeID(t *testing.T) {
```

Comment:
```suggestion
func TestIssue12535(t *testing.T) {
```
This is a very specific scenario of testing: add negative lease, restart node, revoke. In this situation I think it's better to refer to issue.